examples/index.html

```diff
@@ -137,6 +137,17 @@ <h3><a href='./planar_vector_tiles.html' target='_blank'>Plane mode
             <img src='screenshots/planar_vector_tiles.jpg'>
             <iframe src="" style="display: none;"></iframe>
         </div>
+        <div class="exampleContainer">
+            <h3><a href='./orientation_parser.html' target='_blank'>Orientation parser</a></h3>
+            <img src='screenshots/orientation_parser.jpg'>
+            <iframe src="" style="display: none;"></iframe>
+        </div>
+        </div>
+        <div class="exampleContainer">
```

Comment:
I don't feel that we should always add a globe and a planar example everytime we do a feature. What about keeping only the globe one ?

Reply:
The globe example needs the local frame rotation (using `setENUFromGeodesicNormal`), and the planar does not. 2 examples show that it supports both cases. With one example, we need tests for the 2 cases.

Reply:
Ok my bad I didn't read the planar example at all, I assumed it was the same one as the globe !